sdk/ml/azure-ai-ml/azure/ai/ml/entities/_load_functions.py

```diff
@@ -221,6 +221,15 @@ def load_workspace(
 
     :return: Loaded workspace object.
     :rtype: Workspace
+
+    .. admonition:: Example:
+
+        .. literalinclude:: ../../../../samples/ml_samples_workspace.py
```

Comment:
Can you update all of these literalinclude sections to only be one directory removed (i.e. ../samples/ml_samples_workspace.py)?